### PR TITLE
feat(UH-011): add functional delete button

### PR DESCRIPTION
**Pull request by AlejandroSalas1143**
image
image

### CODE REVIEW

File: src/features/opportunites/pages/OpportuniesList/DeleteConfirmation.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export function DeleteConfirmationDialog({ open, onClose, onConfirm }: DeleteConfirmationDialogProps) {
+  return (
+    <BaseModal open={open} onClose={onClose}>
+      <ModalHeader title="Confirm Deletion" onClose={onClose} />
+      <Typography sx={{ mt: 2 }}>
+        Are you sure you want to delete this item?
+      </Typography>
+      <Box sx={{ display: "flex", justifyContent: "flex-end", mt: 3 }}>
+        <Button onClick={onClose}>Cancel</Button>
+        <Button onClick={onConfirm} color="error">
```
Comment (dgop92) [inline src/features/opportunites/pages/OpportuniesList/DeleteConfirmation.tsx]: You can use somthing like this to make the modal more responsive

<BaseModal extraBaseStyles={{ maxWidth: "900px", width: "95%" }}  />

Note: configure the maxWdith base on your requirements

File: src/features/opportunites/pages/OpportuniesList/OpportunitiesTable.tsx [hunk 2]
```diff
@@ -5,18 +5,41 @@ import { DataGrid, GridActionsCellItem, GridColDef } from "@mui/x-data-grid";
...
+
+    setOpenDeleteDialog(false);
+    setSelectedId(null);
+
+    refetch();
+  } catch (error) {
+    console.error("Error deleting opportunity:", error);
+  }
+};
+
```
Comment (dgop92) [inline src/features/opportunites/pages/OpportuniesList/OpportunitiesTable.tsx]: Normally, delete is a mutation as well, so you should use useMutation,

**Review by dgop92 — CHANGES_REQUESTED** [review]
look the suggestions

**Review by dgop92 — APPROVED** [review]
ok

**Review by TheHoodini — APPROVED** [review]
Excellent

**Review by ndelarans — APPROVED** [review]
Ok,  looks good

**Pull request merged**